.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "superbuild"]
+	path = superbuild
+	url = https://gitlab.kitware.com/paraview/common-superbuild.git
```

Comment:
We should mirror common-superbuild to f3d-app/common-superbuild

Reply:
Why ?